config.ts

```diff
@@ -15,21 +26,29 @@ export let config = {
     : [],
   host: envs.HOST || "localhost",
   port: Number(envs.PORT) || 8000,
-  https: !!envs.HTTPS || false,
-  jwt: !!envs.JWT || false,
+  https: Boolean(envs.HTTPS) || false,
+  jwt: Boolean(envs.JWT) || false,
   refreshTokenDays: Number(envs.REFRESH_TOKEN_DAYS) || 120,
   serverCodeLength: Number(envs.SERVER_CODE_LENGTH) || 256,
   sslCertificateLocation: envs.HTTPS_CERTIFICATE_LOCATION,
   sslKeyLocation: envs.HTTPS_KEY_LOCATION,
-  enableTrafficLogs: envs.ENABLE_TRAFFIC_LOGS || false,
-  version: "0.11.5"
+  enableTrafficLogs: Boolean(envs.ENABLE_TRAFFIC_LOGS) || false,
+  version: "0.11.5",
+  link: "",
+  proxy: Boolean(envs.PROXY) || false,
+  proxyCount: Number(envs.PROXY_COUNT) || 0,
```

Comment:
Yes, proxyCount will be 2. This check is for the X-Forwarded-For(XFF) header where it will reject the request if the there are more ips than the value specified in proxyCount. For example, in that setup above and assuming that the proxies modify XFF header accordingly, the client will has to pass through two reverse proxies to reach origin.
ELB -> HAProxy -> cakelandish
When the request passes through ELB first, the header will have the requester ip in the XFF header, after passing through HAProxy, the ELB ip and the requester ip will be in the XFF header. Finally, when the request reaches origin, the header will look something like this: `X-Forwarded-For: REQ_IP, ELB`